packages/wonder-blocks-floating/src/util/constants.ts

```diff
@@ -0,0 +1,3 @@
+// Floating only supports pixel values for the arrow size.
```

Comment:
I confirmed the arrow looks fine with px values when zoomed in at 400%! 

<img width="1722" height="958" alt="Image" src="https://github.com/user-attachments/assets/14dd7679-9b96-471f-a7b7-9dfa335ea579" />